regression/strings-smoke-tests/java_append_int/test.desc

```diff
@@ -1,4 +1,4 @@
-CORE
+FUTURE
 test_append_int.class
 --refine-strings
```

Comment:
I understand that `--string-max-length something` will become the default on test-gen. However, on cbmc itself, the option should not be necessary (because we do not need to create real counterexamples). If anything, the option should be removed in the test descriptions where it is present, but that is not a priority I think.